example/auth/README.md

```diff
@@ -0,0 +1,143 @@
+# Auth Helper Example
+
+This repository contains an example Go application that demonstrates how to use an authentication helper to manage access and refresh tokens for making authenticated API calls easier. The `main.go` script logs in, checks the validity of tokens, and refreshes the token periodically using a background goroutine.
+
+## Prerequisites
+
+- Go 1.15+ installed on your machine.
+- Environment variables for your username and password stored in a `.env` file.
+
+## Setup
+
+### 1. Clone the Repository
+
+```sh
+git clone https://github.com/wikimedia-enterprise/wme-sdk-go.git
+cd wme-sdk-go
+```
+
+### 2. Environment Variables
+
+There is a sample environment file `sample.env` provided in the repository. Rename it to `.env` and add your username and password.
+
+```sh
+mv sample.env .env
+```
+
+Edit the `.env` file to include your credentials:
```

Comment:
Let's remove the `sample.env` from the repo.
Instead add the instruction:
Create a `.env` file with the following contents:

Reply:
Done